index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Barbara Medina's Portfolio</title>
+        <meta charset="utf-8">
+        <meta name= "author" content="Barbara">
+    </head>
+    <body>
```

Comment:
Try to maintain HTML structure sintax using tags header/main/footer to build your section's

Reply:
I am going to following something similar to this. https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Structuring_documents 
I will add a footer at the bottom and have it say copyright made by me. 
And I will add a main before the body. We can also discuss if this is correct either at the next review or at a meeting I schedule with you.